### PR TITLE
Add deployment instructions to README

### PR DESCRIPTION
**Pull request by vilsbole**
## Description
Add instructions on how to deploy new package.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,3 +32,23 @@ Our designs use the [Lato](http://www.latofonts.com/) and [Roboto Mono](https://
 - Use [typeface-lato](https://www.npmjs.com/package/typeface-lato) and [typeface-roboto-mono](https://www.npmjs.com/package/typeface-roboto-mono) to self-host your typefaces when using npm/yarn with Webpack or any other build tool with CSS and font loaders ([instructions](https://github.com/KyleAMathews/typefaces#how))
 - Use [Google Fonts](https://fonts.google.com/?selection.family=Lato:400,700,900|Roboto+Mono) to load the fonts from a CDN (over the Internet) without any configuration (note that [Google collects some usage data](https://developers.google.com/fonts/faq#what_does_using_the_google_fonts_api_mean_for_the_privacy_of_my_users))
 - Download [Lato](http://www.latofonts.com/) and [Roboto Mono](https://fonts.google.com/specimen/Roboto+Mono) directly if you need more control over font loading or if you only plan on using the fonts locally
+
+### Development
```
Comment (Mrtenz) [inline README.md]: `##`

File: README.md [hunk 2]
```diff
@@ -32,3 +32,23 @@ Our designs use the [Lato](http://www.latofonts.com/) and [Roboto Mono](https://
...
+
+You can launch the storybook simply by running `yarn start`
+
+### Deployment
```
Comment (Mrtenz) [inline README.md]: `##`?

File: README.md [hunk 3]
```diff
@@ -32,3 +32,23 @@ Our designs use the [Lato](http://www.latofonts.com/) and [Roboto Mono](https://
...
+
+The npm package is automatically updated by Travis whenever a new tag is pushed to the remote.
+To facilitate version management we use [yarn version](https://yarnpkg.com/lang/en/docs/cli/version/) this step To release a new version you just need to run:
```
Comment (Mrtenz) [inline README.md]: > ... we use yarn version this step To release ...

Should probably be something like "... we use yarn version. To ..."

File: README.md [hunk 4]
```diff
@@ -32,3 +32,23 @@ Our designs use the [Lato](http://www.latofonts.com/) and [Roboto Mono](https://
...
 - Use [Google Fonts](https://fonts.google.com/?selection.family=Lato:400,700,900|Roboto+Mono) to load the fonts from a CDN (over the Internet) without any configuration (note that [Google collects some usage data](https://developers.google.com/fonts/faq#what_does_using_the_google_fonts_api_mean_for_the_privacy_of_my_users))
 - Download [Lato](http://www.latofonts.com/) and [Roboto Mono](https://fonts.google.com/specimen/Roboto+Mono) directly if you need more control over font loading or if you only plan on using the fonts locally
+
+### Development
+
+You can launch the storybook simply by running `yarn start`
+
+### Deployment
+
+The npm package is automatically updated by Travis whenever a new tag is pushed to the remote.
```
Comment (Mrtenz) [inline README.md]: > to the remote

I'd probably change this to something like "to GitHub".

File: README.md (hunk 2 above)
Comment (vilsbole) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (vilsbole) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (vilsbole) [reply README.md]: added explicit reference to Github

File: README.md (hunk 3 above)
Comment (vilsbole) [reply README.md]: good catch!

**Pull request merged**